ansible/roles/schulcloud-server-core/templates/tldraw-delete-files-cronjob.yml.j2

```
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-delete-files-cronjob
+    cronjob: tldraw-delete-files
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: tldraw-delete-files
+    app.kubernetes.io/component: files
```

For the component in this case is tldraw better as value
```suggestion
    app.kubernetes.io/component: tldraw
```